index.html

```diff
@@ -2795,8 +2826,7 @@ <h3><dfn>Writing or pushing content</dfn></h3>
                 </dl>
               </li>
               <li>
-                If |message| is of type {{NDEFMessageInit}} and its
-                |id:string| is not `null`:
+                If |record|'s |id| is not `null`:
```

Comment:
We'd better say _If |record|'s |id| is not `undefined`:_? Given that `NDEFRecordInit` is a dictionary and `id` is non-nullable.

Reply:
You're absolutely right.